### DISCUSSION TITLE
Illuminazione content

### DISCUSSION THREAD

**Discussion by dleone81** [dleone81/apcelettro#42] [General]
Mi serve avere il contenuto della pagina

**Comment by fabiocrazycurls** [comment]
Cosa intendi per contenuto?

**Comment by dleone81** [comment]
Il testo che deve essere inserito

**Comment by fabiocrazycurls** [comment]
Dobbiamo crearlo... mi confronto con Andrea per i testi da creare

**Comment by dleone81** [comment]
Ok, non appena creato caricatelo qui

**Comment by dleone81** [comment]
Ciao @Cattalek93 , @fabiocrazycurls  ho rimosso il contenuto dal menu, quando sarà pronto il contenuto lo re-inserirò
